### PR TITLE
Rke2 march releases

### PR DESCRIPTION
**Pull request by briandowns**

### CODE REVIEW

File: channels-rke2.yaml [hunk 1]
```diff
@@ -352,6 +352,24 @@ releases:
...
+      <<: *charts-v1-21-10-rke2r2
+      rke2-cilium:
+        repo: rancher-rke2-charts
+        version: 1.11.201
+      rke2-canal:
+        repo: rancher-rke2-charts
+        version: v3.21.4-build2022031701
+      rke2-coredns:
+        repo: rancher-rke2-charts
+        version: 1.17.000
```
Comment (rancher-max) [inline channels-rke2.yaml:372]:
```suggestion
        version: 1.17.000
      rke2-ingress-nginx:
        repo: rancher-rke2-charts
        version: 4.0.307
```

File: channels-rke2.yaml [hunk 2]
```diff
@@ -424,6 +442,24 @@ releases:
...
+      <<: *charts-v1-22-7-rke2r2
+      rke2-cilium:
+        repo: rancher-rke2-charts
+        version: 1.11.201
+      rke2-canal:
+        repo: rancher-rke2-charts
+        version: v3.21.4-build2022031701
+      rke2-coredns:
+        repo: rancher-rke2-charts
+        version: 1.17.000
```
Comment (rancher-max) [inline channels-rke2.yaml:465]:
```suggestion
        version: 1.17.000
      rke2-ingress-nginx:
        repo: rancher-rke2-charts
        version: 4.0.307
```

File: channels-rke2.yaml [hunk 3]
```diff
@@ -352,6 +352,24 @@ releases:
       rancher-vsphere-csi:
         repo: rancher-charts
         version: 100.1.0+up2.3.0
+  - version: v1.21.11+rke2r1
+    minChannelServerVersion: v2.6.4-alpha1
```
Comment (rancher-max) [inline channels-rke2.yaml]:
```suggestion
    minChannelServerVersion: v2.6.3-alpha1
```

This is based off a change that was made to dev-v2.6 for the rancher release. Rebasing your branch should pull this in for v1.21.10+rke2r2 so this change is just to make it consistent with that.

File: channels-rke2.yaml [hunk 4]
```diff
@@ -442,3 +478,27 @@ releases:
...
+        version: v3.21.4-build2022031701
+      rancher-vsphere-cpi:
+        repo: rancher-rke2-charts
+        version: 1.1.002
+      rancher-vsphere-csi:
+        repo: rancher-rke2-charts
+        version: 2.5.0-rancher101
+      rke2-coredns:
+        repo: rancher-rke2-charts
+        version: 1.17.000
```
Comment (rancher-max) [inline channels-rke2.yaml:513]:
```suggestion
        version: 1.17.000
      rke2-ingress-nginx:
        repo: rancher-rke2-charts
        version: 4.0.307
```

**Review by rancher-max — CHANGES_REQUESTED** [review]
Made some suggestions for additions based on:
- https://github.com/rancher/rke2/blame/v1.21.11%2Brke2r1/Dockerfile#L98-L117
- https://github.com/rancher/rke2/blame/v1.22.8%2Brke2r1/Dockerfile#L99-L116
- https://github.com/rancher/rke2/blame/v1.23.5%2Brke2r1/Dockerfile#L101-L118

**Review by rancher-max — APPROVED** [review]

**Review by dereknola — APPROVED** [review]

**Review by cwayne18 — APPROVED** [review]

**Pull request merged**